Purpose: Don't include unused signalfd header
Fixes:   Compilation on platforms not having <sys/signalfd.h>
Authur:  Roger Leigh
Status:  Not yet submitted upstream
---
--- a/startpar/startpar.c	2012-01-27 23:20:26.713489556 +0000
+++ b/startpar/startpar.c	2012-01-27 23:20:38.353642868 +0000
@@ -42,7 +42,6 @@
 #include <sys/sysinfo.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
-#include <sys/signalfd.h>
 #ifdef CHECK_FORDEVPTS
 # include <sys/vfs.h>
 #endif
